cmd/gossamer/flags.go

```diff
@@ -77,10 +77,22 @@ var (
 
 // Initialization-only flags
 var (
-	// GenesisFlag Path to genesis JSON file
+	// GenesisRawFlag Path to raw genesis JSON file
+	GenesisRawFlag = cli.StringFlag{
+		Name:  "genesis-raw",
+		Usage: "Path to raw genesis JSON file",
+	}
+)
+
+// BuildSpec-only flags
+var (
+	RawFlag = cli.BoolFlag{
+		Name:  "raw",
+		Usage: "Output as raw genesis JSON",
+	}
```

Comment:
what is this used for? is there another option that build-spec can output?

Reply:
see below.